### DISCUSSION TITLE
Summary Information Stream Custom property

### DISCUSSION THREAD

**Discussion by TheDarkDefenderr** [wixtoolset/issues#7690] [Questions]
Hello,

I would like to add custom Property in the Description of my .msi and .exe file.
So, is it possible to have after right-click on the file (after build), selecting Properties my custom Property (like Title, Subject, etc.) there? How to do that?

Best regards,
Bartosz

**Comment by robmen** [comment]
See the [SummaryInformation element](https://wixtoolset.org/docs/schema/wxs/summaryinformation/).
